releasenotes/notes/runtime-security-agent-fe14b8fab5b6ee9e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The new Runtime Security Agent collects file integrity monitoring events.
```

Comment:
let's mention here:

* that it's disabled by default
* that it only supports Linux (for now?)
* whether it'll be in beta or stable (the release note can be updated later if that's not something that can be publicly documented yet)
* how users can enable it (or a link to public docs? Again the release note can be updated later if that's not public yet)

Reply:
Updated the release note. Regarding whether it's beta or stable and the documentation, we'll update it later it's it ok as all details are not settled yet.